app/bundles/CoreBundle/Assets/js/14.notifications.js

```diff
@@ -0,0 +1,19 @@
+Mautic.notificationIndexLoad = function (translations) { 
+    mQuery(document).ready(function() { 
+        // Hide the New button
+        mQuery("a[href$='account/notifications/new']").hide();
+
+        // Mark a notification as read.
+        mQuery('#notificationTable a')
+            .each(function(index) { 
```

Comment:
Jquery allows you to mass bind events to arrays of elements.

so you don't need the .each AFAIK.